content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -47,13 +47,13 @@ The components marked as **X** are installed within each profile:
 | Core components | | | | | |
 | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-egressgateway` | | X | | |
 | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-ingressgateway` | X | X | | |
-| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-pilot` | X | X | X | |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istiod` | X | X | X | |
 | Addons | | | | | |
 | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`grafana` | | X | | |
 | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-tracing` | | X | | |
 | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`kiali` | | X | | |
 | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`prometheus` | X | X | | X |
 
-To further customize Istio and install addons, you can add one or more `--set <key>=<value>` options in the
-`istioctl manifest` command that you use when installing Istio.
-Refer to [customizing the configuration](/docs/setup/install/istioctl/#customizing-the-configuration) for details.
+To further [customize Istio and install addons](/docs/setup/install/istioctl/#customizing-the-configuration), you can add one or more `--set <key>=<value>` options in the
```

Comment:
I believe this is being deprecated in 1.7 so I'm not sure we want it in master. Maybe this should just go in the release-1.6 branch?

cc @howardjohn

Reply:
the addons will exist in 1.7 still, just not the recommended/default so I think its fine. In 1.8 we will remove it